src/WebJobs.Script.WebHost/Diagnostics/SystemTraceWriter.cs

```diff
@@ -71,9 +75,32 @@ public override void Trace(TraceEvent traceEvent)
             if (string.IsNullOrEmpty(details) && traceEvent.Exception != null)
             {
                 details = Sanitizer.Sanitize(traceEvent.Exception.ToFormattedString());
+                if (
+                    (traceEvent.Exception is FunctionInvocationException ||
+                    traceEvent.Exception is AggregateException ||
+                    traceEvent.Exception is TargetInvocationException) &&
+                    traceEvent.Exception.InnerException != null)
+                {
+                    GetExceptionDetails(traceEvent.Exception.InnerException, out exceptionType, out exceptionMessage, out exceptionStackTrace);
+                }
```

Comment:
I'm wondering whether we should just make it a rule that we want the exception columns to be populated with the inner most exception, regardless of type?

Reply:
Will update to look for the inner most exception.